RealEstate/urls.py

```diff
@@ -19,5 +19,8 @@
 
 urlpatterns = [
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^$', include('RealEstate.apps.core.urls')),
+    url(r'^$', include('RealEstate.apps.core.urls'), name='home'),
+    url(r'^login/$', 'django.contrib.auth.views.login', name='auth_login'),
+    url(r'^logout/$', 'django.contrib.auth.views.logout', {'next_page': '/'}, name='auth_logout'),
+    url(r'^logout/(?P<next_page>)/$', 'django.contrib.auth.views.logout', name='auth_logout_next'),
```

Comment:
This isn't actually used anywhere. I'm going to remove this line. I don't see a login_the_logout view, but I do see the function. Rather than implement my own view, I'll modify the line above so that 'next_page' appropriately is redirected to login.